### PR TITLE
test(ivy): add JIRA references for root-casuse TestBed failures

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
This PR enables more of the `@angular/core` tests running with ivy on `TestBed`.  It basically do 2 things:
* moves `fixmeIvy` from `describe` to `it` to enable more tests running on CI (for cases where there were passing tests but there were not running since the entire `describe` was disabled);
* adds root cause analysis where one can be derived from the error message (there are still many cases where investigation is needed to actually understand the root cause)

### CODE REVIEW

File: packages/core/test/linker/regression_integration_spec.ts [hunk 1]
```diff
@@ -160,13 +162,14 @@ function declareTests(config?: {useJit: boolean}) {
         expect(injector.get(token)).toEqual(tokenValue);
       });
 
-      it('should support providers with string token with a `.` in it', () => {
-        const token = 'a.b';
-        const tokenValue = 1;
-        const injector = createInjector([{provide: token, useValue: tokenValue}]);
+      fixmeIvy('FW-646: Cannot create property __NG_ELEMENT_ID__ on a primitive type') &&
```
Comment (kara) [inline packages/core/test/linker/regression_integration_spec.ts]: Can you change so the reason matches the JIRA ticket title?

> FW-646: Directive providers don't support primitive types as DI tokens

**Review by kara — COMMENTED** [review]
Looks good except for these nits. 

p.s. also noticed a typo in the commit message. fix?

File: packages/core/test/linker/ng_container_integration_spec.ts [hunk 2]
```diff
@@ -15,10 +15,11 @@ import {fixmeIvy} from '@angular/private/testing';
 
 {
   if (ivyEnabled) {
-    fixmeIvy('unknown') && describe('ivy', () => { declareTests(); });
+    fixmeIvy('FW-663: ReferenceError: goog is not defined') &&
```
Comment (AndrewKushnir) [inline packages/core/test/linker/ng_container_integration_spec.ts]: @pkozlowski-opensource I'd propose to move this `fixmeIvy` next to the `should support the "i18n" attribute` test (which is causing this error), the rest of the tests should not have this issue and can be investigated further for other failures.

Comment (pkozlowski-opensource) [reply packages/core/test/linker/ng_container_integration_spec.ts]: Done. You are right, `goog is not defined` issue is blocking just one test here.

File: packages/core/test/linker/integration_spec.ts [hunk 3]
```diff
@@ -1412,7 +1434,7 @@ function declareTests(config?: {useJit: boolean}) {
         expect(getDOM().querySelectorAll(fixture.nativeElement, 'script').length).toEqual(0);
       });
 
-      it('should throw when using directives without selector', () => {
+      fixmeIvy('pull/27169') && it('should throw when using directives without selector', () => {
```
[the remark this answers is not in the document]
Comment (pkozlowski-opensource) [reply packages/core/test/linker/integration_spec.ts]: Done

File: packages/core/test/linker/query_integration_spec.ts [hunk 4]
```diff
@@ -15,627 +15,1608 @@ import {Subject} from 'rxjs';
...
-      it('should remove manually projected templates if their parent view is destroyed', () => {
-        const template = `
+  fixmeIvy('FW-670: Internal Error: The name q is already defined in scope') && describe(
+                                                                                    'Query API',
+                                                                                    () => {
+
+                                                                                      beforeEach(
+                                                                                          () =>
+                                                                                              TestBed
+                                                                                                  .configureTestingModule({
```
[the remark this answers is not in the document]
Comment (pkozlowski-opensource) [reply packages/core/test/linker/query_integration_spec.ts]: Agreed, all credits go to clang :-) 

Exceptionally, for this particular one, I've just added ticket reference (`FW-670`) and the full ticket title in a comment above.

**Review by mhevery — APPROVED** [review]

**Pull request closed without merging**